app/controllers/requests/request_controller.rb

```diff
@@ -65,6 +66,23 @@ def generate
       # flash.now[:notice] = "You are eligible to request this item. This form is in development and DOES not submit requests yet."
     end
 
+    def ill_to_edd_form
+      render layout: false
+      flash.now[:notice] = "Please Supply a valid Transaction Number."
+    end
+
+    def ill_to_edd
+      @ill_to_edds = Requests::IllToEdd.new(params)
+
+      if @ill_to_edds.errors.any?
+        response = @ill_to_edds.errors.to_json
+      else
+        response = @ill_to_edds.response.body.encode("UTF-8", invalid: :replace, undef: :replace).to_json
+      end
+
+      render json: response
```

Comment:
Since we are building a simple form for the initial implementation of this we should have this behave more like a "normal" webform and return a response that comes back as a rails view. I was thinking that ILLiad itself might call this as an API and act upon the JSON response that currently comes back from ill_to_edd. However it seems like that is going to take some time to work out so we should just get this fallback form implemented as soon as we can.